### PR TITLE
DM-35159: Add utility functions for remaking plots

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: python/lsst/rubintv/production/utils.py [hunk 1]
```diff
@@ -72,3 +81,148 @@ def checkRubinTvExternalPackages(exitIfNotFound=True, logger=None):
...
+    bucket : `google.cloud.storage.bucket.Bucket`, optional
+        The GCS bucket, created if not supplied.
+
+    Returns
+    -------
+    seqNums : `list` [`int`]
+        Sorted list of ints of the seqNums for which the specified plot exists.
+    """
+    if channel not in CHANNELS:
+        raise ValueError(f"Channel {channel} not in {CHANNELS}.")
```
Comment (czwa) [inline python/lsst/rubintv/production/utils.py:105]: This `raise` isn't discussed in the docstring.

File: python/lsst/rubintv/production/utils.py [hunk 2]
```diff
@@ -72,3 +81,148 @@ def checkRubinTvExternalPackages(exitIfNotFound=True, logger=None):
...
+        The lsst.rubintv.production Channel object.
+
+    Raises
+    ------
+    ValueError:
+        Raised if the channel is unknown, or creating by name is not supported
+        for the channel in question.
+    """
+    if channel not in CHANNELS:
+        raise RuntimeError(f"Channel {channel} not in {CHANNELS}.")
```
Comment (czwa) [inline python/lsst/rubintv/production/utils.py:142]: Same as above.

File: python/lsst/rubintv/production/utils.py [hunk 3]
```diff
@@ -72,3 +81,148 @@ def checkRubinTvExternalPackages(exitIfNotFound=True, logger=None):
...
+        Remake all plots, regardless of whether they already exist in the
+        bucket?
+    notebook : `bool`, optional
+        Is the code being run from within a notebook? Needed to correctly nest
+        asyncio event loops in notebook-type environments.
+    """
+    from google.cloud import storage
+
+    if channel not in CHANNELS:
+        raise RuntimeError(f"Channel {channel} not in {CHANNELS}")
```
Comment (czwa) [inline python/lsst/rubintv/production/utils.py:200]: As above.

**Review by czwa — APPROVED** [review]
The only problem I can see are some missing docstring elements.

File: python/lsst/rubintv/production/utils.py (hunk 1 above)
Comment (mfisherlevine) [reply python/lsst/rubintv/production/utils.py:105]: Thanks - I've added all of these, changed them all to being `ValueError`s, and made all the docs actually agree with the `raise` too, which I just spotted they didn't before 🙂

**Pull request merged**